src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.cs

```diff
@@ -173,15 +173,12 @@ await semanticDocument.WithSyntaxRootAsync(result.Root, cancellationToken).Confi
         protected override AbstractFormattingRule GetCustomFormattingRule(Document document)
             => FormattingRule.Instance;
 
-        protected override SyntaxToken? GetInvocationNameToken(IEnumerable<SyntaxToken> methodNames)
-            => methodNames.FirstOrNull(t => !t.Parent.IsKind(SyntaxKind.MethodDeclaration));
```

Comment:
this code exists from a time when we had multiple tokens annotated, both for the invocation site and the declaration side.  this code then filtered down to the one wanted.  however, this was changed and we now have a single annotation for this single location.  so we can jsut fetch it without needing to do this.